checks/ensure_packet_redirect_disabled/user-feedback-text.md

```diff
@@ -0,0 +1,23 @@
+# Failing Check: Packet redirect sending is disabled
+Reason: Packet redirect sending is disabled
+
+## Why is this a Problem?
+ICMP Redirects are used to send routing information to other hosts. As a host itself does not act as a router (in a host only configuration), there is no need to send redirects.
+
+An attacker could use a compromised host to send invalid ICMP redirects to other router devices in an attempt to corrupt routing and have users access a system set up by the attacker as opposed to a valid system.
+
+Further reading can be found at CIS Ubuntu 18.04 Benchmark v3.1, section 3.1.1.
```

Comment:
Er dette dokumentet tilgjengelig for "folk flest"? Kunne vi hatt en link til det?

Reply:
Eh, vet ikke? Referansen er der på direkte forespørsel fra Kolide.

Tviler på at gemene hop som bruker naisdevice bryr seg nok til at det er verdt å finne en lenke og holde den oppdatert.